docs/directives/needuml.rst

```diff
@@ -345,6 +345,32 @@ Finds a list of Sphinx-Need objects that pass the given filter string.
       {% endfor %}
 
 
+.. _needuml_jinja_ref
+
+ref(id, content, text)
```

Comment:
Sorry, to bother you here again.
I have thought about ``content`` and I think it's not 100% clear what it means, as it describes what it is used for and not what the source of the data is.
Can we rename it to ``option``, as all need-values in your documentation are referenced as ``option``?